CMakeLists.txt

```diff
@@ -119,6 +121,14 @@ find_package(Qt6 REQUIRED COMPONENTS ${QT_FPDEPS})
 qt_standard_project_setup(REQUIRES 6.6)
 set(QT_QML_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/qml_modules)
 
+if (INSTALL_QML_LIB)
+	install(
+		DIRECTORY ${CMAKE_BINARY_DIR}/qml_modules/
+    	DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/qt-6/qml
```

Comment:
1. Not sure but there might be a more specific cmake prefix for /lib if its at a different path on some systems
2. match the indentation of the previous line

Reply:
Ah yeah, that would be CMAKE_INSTALL_LIBDIR, I forgot about that one. Nydragon will fix